fsw/src/sample_app.h

```diff
@@ -37,25 +37,42 @@
 #include "cfe_evs.h"
 #include "cfe_sb.h"
 #include "cfe_es.h"
-#include "cfe_tbl_filedef.h"
-#include "cfe_tbl_task.h"
 #include "cfe_tbl_internal.h"
 
 #include "sample_app_perfids.h"
 #include "sample_app_msgids.h"
 #include "sample_app_msg.h"
-#include "sample_app_events.h"
-#include "sample_app_version.h"
```

Comment:
*_events.h and *_version.h follow the app development model, I recommend against removing them.  Typically define these values in individual header files such that changes to logic are more obvious (vs changes to id's or version numbers).

Reply:
Resolved, follows app dev guide model